src/swatiShinde30/assignment3/Question12.java

```diff
@@ -0,0 +1,35 @@
+package swatiShinde30.assignment3;
+
+import java.util.Scanner;
+
+/*
+ * WAP to find word is palindrome or not ? 
+ */
+public class Question12 {
+	static void palindrom(String str) {
+		String str2 = "";
+		boolean flag = false;
+		for (int i = str.length() - 1; i >= 0; i--) {
+			char ch = str.charAt(i);
+			str2 = str2 + ch;
+		}
+		System.out.println(str2);
+		for (int j = 0; j < str.length(); j++) {
```

Comment:
This is wrongly written. What if only one character matches and others are not matching .. still it will set flag as true.
Try input : MaulikM.
It will say this is palindrom